Bangazon.Tests/Bangazon_ProductManagerShould.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace BangazonCLI.Tests
+{
+    // Written by: Chaz Henricks
+    // Tests Functionality of Product manager; adds new Product and gets a list of Products.
+    public class ProductManagerShould : IDisposable
+    {
+        private readonly ProductManager _cm;
+        private readonly dbUtilities _db;
+        // Creates a Product manager and connection with the database..
+        
+        public ProductManagerShould ()
+        {
+            _db = new dbUtilities("BANGAZONCLI_TEST_DB");
+            _cm = new ProductManager(_db);
+            _db.CheckProduct();
+        }
+        // Tests to see if Products are really being added by our methods.
+        [Fact]
+        public void AddNewProduct()
+        {
+            Product newProduct = new Product();
+            
+                newProduct.Name = "Ball"; 
+                newProduct.Description= "Its a ball"; 
+                newProduct.Price = 9000; 
+                newProduct.DateCreated= DateTime.Today; 
+
+            
+
+        var result = _cm.AddNewProduct(newProduct);
+
+            Assert.True(result !=0);
+        }
+        // Tests to see if our methods are getting a list of Products.
+        [Fact]
+        public void ListProducts()
+        {
+
+            List<Product> ProductList = _cm.GetProductList();
+
+            Product newProduct = new Product();
```

Comment:
For this first test its just to add a product to the list. 
It will end up as a theory that will post X amount of products for testing - but a product needs to be created in order to add it to the DB eventually.